lib/install/management/finder.go

```diff
@@ -269,8 +269,11 @@ func (d *Dispatcher) searchVCHsPerRP(pool *object.ResourcePool) ([]*vm.VirtualMa
 	var pools []*object.ResourcePool
 	pools = append(pools, pool)
 
-	children, err := d.listChildrenPools(pool)
+	children, err := d.session.Finder.ResourcePoolList(d.ctx, path.Join(pool.InventoryPath, "..."))
```

Comment:
@dougm just curious. As while I'm working on this issue, found that https://github.com/vmware/vic/pull/4345/commits/c8e4212375555d72d24c244c708022d845bd4e64#diff-6a167380a66d015ed8093337bcbaa6f9R248 will return two resources, that's why we have duplicate result. Is that fixed in govmomi?